HttpSignature.md

```diff
@@ -237,10 +234,12 @@ App                                   Server                Doc             Cred
 Steps (4) and (5), where the server retrives a (cached)copy of the key, are as before. 
 
 Steps (6) can be run in parallel with (4) to fetch the Credential document.
-This also can be cached.
-If the URL is relative it will be found on Resource Server.
-If the URL is remote it may need to be fetched.
-If the URL is enclosed in `>` and `<` and a P2P extension is available the server can request the credential from the client directly using the same connection opened in (3).
+This also can be cached. 
+If the `Credential` header URL
+* is enclosed in '<' and '>' then 
+  * if it is relative it can be fetched on the same server
+  * if is is an absolute URL it can be fetched by opening a new connection
+* is enclosed in `>` and `<` and P2P HTTP extension is enabled, the server can request the credential from the client directly using the same connection opened in (3).
```

Comment:
As above, I think this probably should be
```suggestion
* is enclosed in `<` and `>` and P2P HTTP extension is enabled, the server can request the credential from the client directly using the same connection opened in (3).
```

Reply:
Same point as above. Perhaps this is such a new idea that it requires more elaboration, perhaps even a picture or something.

Reply:
I think it requires a lexical/literal example, at minimum.  If there are other wrapping characters -- whether brackets, braces, parentheses, quotation marks, or otherwise -- which may be used in a more typical manner (the only thing I can quickly think of that uses `>…<` wrappers in that order is an ASCII emoticon for a cat, `>^..^<`), I would suggest using them instead.

Reply:
The way I see it is in terms of indexicals eg: I and you. In an HTTP connection of a client `C` to a server `example.com`, the *you* for the client is `example.com`. The server on the other hand knows nothing of the client so it only can say `you`. The client wishes to pass a key to the server but it can say: the key at my house at address so and so, or the key in your pocket, or the key in my pocket. We need to be able to distinguish between "the key in my pocket" and "the key in your pocket".

Reply:
Making such distinction doesn't seem to require any specific wrapping characters. It requires knowing who's "speaking", and to whom -- as the entity to which the possessive "my" or "your" refers changes in meaning with the speaker and the audience.  In HTTP, we can speak of the Client and the Server, and describe what action each might be taking (requesting, responding, etc.). In other spheres, we have conventions like Requesting Party, Relying Party, Responding Party, etc., to serve the same purpose.

Sometimes, it's very hard to describe an interaction with generalities, but trivial to describe it through a concretized example.  I think this might be one of those situations, at least at this point in the ecosystem's development.

Reply:
> Making such distinction doesn't seem to require any specific wrapping characters. It requires knowing who's "speaking", and to whom -- as the entity to which the possessive "my" or "your" refers changes in meaning with the speaker and the audience.

Let us say we start with a connection from Phone to Pod .
Phone requests a secure resource, is returned a 401 by Pod, and then sends a signed message with a relative URL to the Credential. We want to allow the following two possibilities to exist:
 
 1. the Credential is on the Phone at >/auth/Cred1<
 2. the Credential is on the server at </auth/Cred1>
 
The relative path `/auth/Cred1` is the same in both cases, so we use the angel bracket reversal to distinguish on which side the URL is relative in that conversation.

Reply:
I wrote more about what P2P opens up to the web here https://github.com/w3c/architecture/issues/14 
I still want to integrate this into the spec here, but need to be careful not to make too much of it, as what is proposed can also work without it.